test/unit/extension/lib/getAuthenticatedCheckoutUrlTest.js

```diff
@@ -0,0 +1,38 @@
+const assert = require('assert')
+
+const getAuthenticatedCheckoutUrl = require('../../../../extension/lib/getAuthenticatedCheckoutUrl')
+
+describe('getAuthenticatedCheckoutUrl - unit', () => {
```

Comment:
Additionally you could assert that the errors are logged

Reply:
I am not sure how to stub or mock the AuthRepository in a way that would cause the getAuthenticatedCheckoutUrl would use it because the AuthRepository is not passed into the getAuthenticatedCheckoutUrl method. Can you show me how to do this?